src/main/drivers/bus.h

```diff
@@ -102,6 +102,8 @@ typedef enum {
     DEVHW_IST8308,
     DEVHW_QMC5883,
     DEVHW_MAG3110,
+    DEVHW_LIS3MDL,
+    DEVHW_LIS3MDL_A2,
```

Comment:
Having two items for the same hardware is not wise. I suggest keeping the one that is compatible with off the shelf modules for Pixhawk.

Reply:
I'm planning to address multiple possible I2C addresses for the same sensor chip.

Reply:
Looks like PX4 is only supporting the address 0x1E.  Maybe it's not even necessary to check for the alternative address?

https://github.com/PX4/Firmware/blob/02d4405a6205cfa050925ac0edcf4d0a13382632/src/drivers/magnetometer/lis3mdl/lis3mdl_i2c.cpp#L62

https://github.com/PX4/Firmware/blob/02d4405a6205cfa050925ac0edcf4d0a13382632/src/drivers/magnetometer/lis3mdl/lis3mdl_i2c.cpp#L88-L92